src/server/status/cgroup.js

```diff
@@ -0,0 +1,136 @@
+import fs from 'fs';
+import { promisify } from 'bluebird';
+import { join as joinPath } from 'path';
+
+// Logic from elasticsearch/core/src/main/java/org/elasticsearch/monitor/os/OsProbe.java
+
+const CONTROL_GROUP_RE = new RegExp('\\d+:([^:,]+(?:,[^:,]+)?):(/.*)');
```

Comment:
do we need to update this to match? https://github.com/elastic/elasticsearch/commit/00a8b8799f6b7de803bc8afeaf7de392f3d0d28c